server/lease/metrics.go

```diff
@@ -49,11 +49,88 @@ var (
 			// 1 second -> 3 months
 			Buckets: prometheus.ExponentialBuckets(1, 2, 24),
 		})
+
+	leaseAttached = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
What's the purpose of metrics about lease attachements? What you are trying to measure? number of keys per lease?

Reply:
> number of keys per lease?

Precisely, yes.

Reply:
@serathius ping^

Reply:
> number of keys per lease?

It can only get the total number of keys attached by all leases because obviously there isn't a `lease_id` label.